plugins/code-climate/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@backstage/plugin-code-climate",
+  "version": "0.1.0",
```

Comment:
New plugins start at `0.0.0` and should instead have a changeset with a `minor` bump